actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -410,6 +410,10 @@ def [](key)
       convert_hashes_to_parameters(key, @parameters[key])
     end
 
+    def to_param
```

Comment:
Why not consider it a bug fix and just delegate it to `to_h`? Far as I can tell this breaks backwardscompatibility intentionally (instead of doing it unintentionally before).